chcemvediet/apps/accounts/forms.py

```diff
@@ -84,23 +85,30 @@ class SettingsForm(forms.Form):
             }),
             )
 
-    help_text = _(u'accounts:SettingsForm:custom_anonymized_strings:help_text')
-    plural_form = plural(WORD_SIZE_MIN, u'1:znak', u'2~4:znaky', u'znakov')
+    help_text = ungettext(
+            u'accounts:SettingsForm:custom_anonymized_strings:help_text',
+            u'accounts:SettingsForm:custom_anonymized_strings:help_text %(count)s.',
+            WORD_SIZE_MIN) % {u'count': WORD_SIZE_MIN}
     custom_anonymized_strings = forms.CharField(
             label=_(u'accounts:SettingsForm:custom_anonymized_strings:label'),
             required=False,
-            help_text=help_text.format(n=WORD_SIZE_MIN, plural_form=plural_form),
+            help_text=help_text,
             widget=forms.Textarea(attrs={
                 u'class': u'pln-autosize chv-visible-if-custom-anonymization',
                 u'cols': u'', u'rows': u'',
                 }),
-            )#
+            )
+
+    error_message = ungettext(
```

Comment:
@martinmacko47 Nerozumiem preco sa `ungettext` sprava zvlastne. Ak je funkcia volana primo v metode tak vzdy vracia len jednu formu t.j. **n znaky**. Ak je definovana mimo metody tak funkguje spravne a vracia mi vsetko forky **n znak/znaky/znakov**.

Reply:
@viliambalaz Netusim, malo by to funovat aj normalne, ked sa `ungettext` vyhodnoti vo vnutri funkcie. Praveze sa to po spravnosti ma vyhodnocovat vo vnutri funkcie, kedze az vtedy vies do akeho jazyka mas lokalizovat. Ked to vyhodnotis takto globalne, tak dostanes jeden string, ktory pouzijes bez ohladu na to, aku ma user nastavenu lokalizaciu.

Skus to prerobit na "python-brace-format". Tj. nieco ako:

```
error_message = ungettext(
  u'accounts:SettingsForm:custom_anonymized_strings:error:line_too_short',
  u'accounts:SettingsForm:custom_anonymized_strings:error:line_too_short {count}',
  WORD_SIZE_MIN).format(count=WORD_SIZE_MIN)
```
a v `django.po`
```
  #, python-brace-format
  msgid "accounts:SettingsForm:custom_anonymized_strings:error:line_too_short"
  msgid_plural "accounts:SettingsForm:custom_anonymized_strings:error:line_too_short {count}"
  msgstr[0] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne {count} znak."
  msgstr[1] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne {count} znaky."
  msgstr[2] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne {count} znakov."
```